notebooks/cytomining-ecosystem/visualize-landscape.py

```diff
@@ -0,0 +1,353 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.15.2
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# # Visualize Cytomining Ecosystem Software Landscape
+#
+# Visualizations related to Cytomining ecosystem software landscape analysis.
+#
+# ## Plot Foci
+#
+# - User base size
+# - Usage
+# - Maturity
+
+# +
+import asyncio
+import os
+import pathlib
+import random
+from datetime import datetime
+
+import nest_asyncio
+import numpy as np
+import pandas as pd
+import plotly.colors as pc
+import plotly.express as px
+import plotly.graph_objects as go
+import plotly.io as pio
+import pytz
+from box import Box
+from itables import to_html_datatable
+from plotly.offline import plot
+from plotly.subplots import make_subplots
+from pyppeteer import launch
+from ydata_profiling import ProfileReport
+
+# set plotly default theme
+pio.templates.default = "simple_white"
+
+# get the current datetime
+tz = pytz.timezone("UTC")
+current_datetime = datetime.now(tz)
+
+# set common str's
+title_prefix = "Cytomining Ecosystem Software Landscape Analysis"
+
+# export locations relative to this notebook
+export_dir = "../../docs/cytomining-ecosystem"
+# -
+
+# read in project metric data
+df_projects = pd.read_parquet("data/project-github-metrics.parquet")
+df_projects = df_projects.reset_index(drop=True)
+df_projects.info()
+
+df_projects["category"] = df_projects["Project Landscape Category"].str[0]
+df_projects.head(5)
+
+# +
+# create a duration for relative comparisons below
+df_projects["Duration Created to Now"] = current_datetime - df_projects["Date Created"]
+
+# create a years count for project time duration
+df_projects["Duration Created to Now in Years"] = (
+    df_projects["Duration Created to Now"].dt.days / 365
+)
+# -
```

Comment:
You may also consider calculating a log transformed axis for star count and visualizaing this instead

Reply:
Ah great point, thanks! I'll add this in.